app/build.gradle

```diff
@@ -33,7 +33,7 @@ dependencies {
     implementation 'com.android.support:support-annotations:28.0.0'
     implementation 'com.android.support.constraint:constraint-layout:1.1.3'
     implementation 'com.google.android.gms:play-services-maps:16.0.0'
-//    testImplementation 'junit:junit:4.12'
+    testImplementation 'junit:junit:4.12'
```

Comment:
Is this supposed to be uncommented?

Reply:
yes, we need it for our junits to run. i tested it; they run and it builds and runs the app successfully still